client/hosting/overview/components/plan-storage-bar.tsx

```diff
@@ -1,24 +1,34 @@
 import { ProgressBar } from '@automattic/components';
 import { SiteMediaStorage } from '@automattic/data-stores';
 import { Icon, cloud } from '@wordpress/icons';
+import clsx from 'clsx';
 import filesize from 'filesize';
 import { useTranslate } from 'i18n-calypso';
 import { FC, PropsWithChildren } from 'react';
 
 interface Props {
 	mediaStorage: SiteMediaStorage;
 }
+const MINIMUM_DISPLAYED_USAGE = 2.5;
+const ALERT_PERCENT = 80;
 
 const PlanStorageBar: FC< PropsWithChildren< Props > > = ( { children, mediaStorage } ) => {
 	const translate = useTranslate();
+	const { storageUsedBytes, maxStorageBytes } = mediaStorage;
 
-	const percent = Math.min(
-		Math.round( ( ( mediaStorage.storageUsedBytes / mediaStorage.maxStorageBytes ) * 1000 ) / 10 ),
-		100
+	// Ensure that the displayed usage is never fully empty to avoid a confusing UI
+	const displayedUsage = Math.max(
+		MINIMUM_DISPLAYED_USAGE,
+		Math.round( ( ( storageUsedBytes / maxStorageBytes ) * 1000 ) / 10 )
 	);
+	const percent = Math.min( displayedUsage, 100 );
```

Comment:
It's not critical, but I propose rewriting it a bit, since at a glance it looks not clear and difficult to read a lot of deep math operations.
What is your opinion about the next?

```suggestion
let usagePercent = Math.round( ( ( storageUsedBytes / maxStorageBytes ) * 1000 ) / 10 );

	// Ensure that the displayed usage is never fully empty to avoid a confusing UI
	usagePercent = Math.max( MINIMUM_DISPLAYED_USAGE, usagePercent );
	// Ensure that somehow it's not bigger than 100
	usagePercent = Math.min( usagePercent, 100 );
```

Reply:
I love it! I thought about breaking up that math too, but sometimes I think I'm over-simplifying and making things more verbose than they need to be. Thanks for helping me dial in on the sweet spot!